### ISSUE TITLE
Move All Fonts to Sass Variables

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#132]

**Comment by martypowell** [comment]
@b3u just noticed, that i never got back to you, ill follow up within the hour.

**Comment by martypowell** [comment]
@b3u looks like this is still valid. The fonts can go in this [file](https://github.com/baltimorecounty/BCPL-assets/blob/integration/stylesheets/partials/core/_fonts.scss). 

Please use a hyphenation in the variable names. for example:

'TTRoundsCondensed-BlackItalic' would be `$TTRoundsCondensed-BlackItalic`.

Once the variables are created, the next task would be to replace any string font family values with those variables.

An example would be here:

https://github.com/baltimorecounty/BCPL-assets/blob/a82e033fe18439833e867c78f238a94e2a357469/stylesheets/partials/units/_typography.scss#L3

Changing to 'Open Sans', Arial, Helvetica, sans-serif; to `use a newly created font variable $default-body-font`.

Sorry, I know this is a very tedious task, if you any questions, please let me know. 

FYI @tmccoy529

**Comment by b3u** [comment]
Does `FontAwesome` become a variable as well?

**Comment by martypowell** [comment]
> Does `FontAwesome` become a variable as well?

Yes, I think that's a good idea as least for now. Unless you have a better suggestion?

**Comment by b3u** [comment]
k
